src/schemas/Helpdesk/jiraRequests.ts

```diff
@@ -154,12 +154,12 @@ export const createJiraClient = async (): Promise<JiraClient> => {
 		type ResponseBody = {
 			values: Array<OrganizationCustomer>;
 		};
-		const response = await fetch(url, {
+		const response = (await fetch(url, {
 			method: 'get',
 			headers: requestHeaders,
 		})
 			.then(restErrorResponseHandler)
-			.then((response) => response.json() as ResponseBody);
+			.then((response) => response.json())) as ResponseBody;
```

Comment:
https://github.com/EUCANCan-Federated-Data-Portal/variant-indexer/blob/main/src/external/song/SongClient.ts#L84

there's a cool way to validate `response.json()` results similar to something Jon has wrote here. Where you can use zod to verify the response.

I realise using `as` is how it's done already - just sharing for knowledge sake

Reply:
`safeParse` ?

Reply:
that function yes, you'd have a zod schema of what you expect the response to be
then use `safeParse` to verify that the response is what you expect
similar to what you'd do if you wrote a typeguard